src/bioregistry/app/templates/base.html

```diff
@@ -52,11 +52,19 @@
             <li class="nav-item active">
                 <a class="nav-link" href="{{ url_for("ui.collections") }}">Collections</a>
             </li>
-            <li class="nav-item active">
-                <a class="nav-link" href="{{ url_for("summary") }}">Summary</a>
-            </li>
-            <li class="nav-item active">
-                <a class="nav-link" href="{{ url_for("download") }}">Downloads</a>
+
+
+            <li class="nav-item active dropdown">
+                <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                    About
+                </a>
+                <div class="dropdown-menu" aria-labelledby="navbarDropdown">
+                    <a class="dropdown-item" href="{{ url_for("summary") }}">Summary</a>
+                    <a class="dropdown-item" href="{{ url_for("download") }}">Downloads</a>
```

Comment:
I think Downloads could warrant its own heading instead of being part of the "About" section.